core/src/main/java/org/elasticsearch/search/aggregations/bucket/terms/TermsAggregatorFactory.java

```diff
@@ -69,8 +69,8 @@ public TermsAggregatorFactory(String name, Type type, ValuesSourceConfig<ValuesS
     @Override
     protected Aggregator createUnmapped(Aggregator parent, List<PipelineAggregator> pipelineAggregators, Map<String, Object> metaData)
             throws IOException {
-        final InternalAggregation aggregation = new UnmappedTerms(name, order, config.format(), bucketCountThresholds.getRequiredSize(),
-                bucketCountThresholds.getShardSize(), bucketCountThresholds.getMinDocCount(), pipelineAggregators, metaData);
+        final InternalAggregation aggregation = new UnmappedTerms(name, order, bucketCountThresholds.getRequiredSize(),
```

Comment:
I'm adding test for this, yeah.

I think this is safe because buckets use the format form their parent internal aggregation. I wonder if instead of trying to make UnmappedTerms work as the reducing aggregation we should instead do like UnmappedSampler and shift to another of the aggregations.